Sources/Commands/SwiftTool.swift

```diff
@@ -762,11 +764,14 @@ private func findPackageRoot() -> AbsolutePath? {
     return root
 }
 
-private func getEnvBuildPath() -> AbsolutePath? {
+/// Returns path of environment build path relative to the directory
+private func getEnvBuildPath(relativeTo buildPath: AbsolutePath?) -> AbsolutePath? {
```

Comment:
This doesn't need to be optional and we can just call it workingDir.

```
private func getEnvBuildPath(workingDir: AbsolutePath?) -> AbsolutePath?
```

Reply:
Done! 😁

Reply:
@aciidb0mb3r just so I can understand better your thinking process, why workingDir instead of buildPath?